### DISCUSSION TITLE
Extracting details of vulnerabilities in Grafana Dashboard

### DISCUSSION THREAD

**Discussion by Brzozova** [aquasecurity/trivy-operator#1973] [Help & Support] [answered]
Hi!
I was able to extract information about the number of vulnerabilities for each image in my cluster with this instruction: https://aquasecurity.github.io/trivy-operator/v0.11.0/tutorials/grafana-dashboard/
Looks great, however that's not enough information to help developers to fix issues, so I would like to expand it.
Is it possible to also extract the details like each vulnerability score, vulnerability ID, fixed version?
I would appreciate any ideas, or examples of how to do it.

**Comment by chen-keinan** [comment, marked as answer]
@Brzozova there are several flags which enable [additional info](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml#L113-L146) for metric which are disable by default. take a look at it.
note each one of it produce a different metric. please also look at [supported metrics structures](https://aquasecurity.github.io/trivy-operator/latest/tutorials/integrations/metrics/)

**Reply by Brzozova** [reply]
This is exactly what I was looking for. Thank you @chen-keinan !!!
